packages/k8s/tests/prepare-job-test.ts

```diff
@@ -62,6 +62,28 @@ describe('Prepare job', () => {
     ).resolves.not.toThrow()
   })
 
+  it('should prepare job with envs CI and GITHUB_ACTIONS', async () => {
```

Comment:
Should we include a test to confirm the outcome if either the `GITHUB_ACTIONS` or the `CI` environment variable already exists in the job definition?